src/modules/hooks/useBlockExplorer.ts

```diff
@@ -0,0 +1,32 @@
+import { useCallback } from 'react';
+import { useChains } from 'wagmi';
+
+type ChainEntityType = 'address' | 'tx' | 'token';
+
+interface IChainEntity {
+    type: ChainEntityType;
+    chainId?: number;
+    id: string;
```

Comment:
Let's add a short description for these parameters and specify that we default the chain to `chains[0]`

Reply:
✅ https://github.com/aragon/ods/pull/220/commits/10f222ff7c19f805283f718a16e7cbb8427199dd